apps/crn-server/src/app.ts

```diff
@@ -468,6 +464,27 @@ export const appFactory = (libs: Libs = {}): Express => {
       'IS_CONTENTFUL_ENABLED_V2',
     );
 
+  featureFlagDependencySwitch.setDependency(
+    'tutorials',
+    libs.tutorialsSquidexDataProvider ||
+      new TutorialsSquidexDataProvider(squidexGraphqlClient),
+    'IS_CONTENTFUL_ENABLED_V2',
+    false,
+  );
+  featureFlagDependencySwitch.setDependency(
+    'tutorials',
+    libs.tutorialsContentfulDataProvider ||
+      new TutorialsContentfulDataProvider(contentfulGraphQLClient),
+    'IS_CONTENTFUL_ENABLED_V2',
+    true,
```

Comment:
It is still necessary to add the header here https://github.com/yldio/asap-hub/blob/master/apps/crn-frontend/src/discover/tutorials/api.ts#L9 so we can see the data coming from Contentful using the `ASAP_CONTENTFUL` cookie.

Reply:
Done.